### ISSUE TITLE
[SpongeForge][Multiplayer]

### ISSUE THREAD

**Issue by Twisted-Code** [rubensworks/EvilCraft#571]
Forge log link:
will retrieve the log for you after I eat dinner
Expected behaviour:
I expected the dark tank to reliably function as a bucket with which I could hold more than 1000mb,, and usually it does... except when it doesn't. 
Problem:
For some reason, I can no longer add or remove blood (or other fluids, though I mainly use it for blood) from my dark tank. Instead of emptying what blood it has into my infuser, for example, it instead opens the GUI as if it's empty, even though it displays blood inside it and will not allow me to add any fluid. Placing it down doesn't fix it, nor does crafting it with another dark tank, and in the latter case, I just get a bigger but equally unusable tank. I've also tried turning on auto supply and holding my blood extractor, but alas, to no avail.
However, it should be noted that I have yet to try restarting the server, though I doubt it'll help.

Tl;dr if I wanted a mundane decoration that looks like a seared window with blood in it[*], I would suggest it.
Steps to reproduce the problem:
- no bucking clue. If I had a clue, I would test in single player to see if I could rule out SpongeForge and for that matter, multiplayer altogether

\[*] Actually might not be a bad idea to add craftable decoration blocks of every functional block like how EnderIO does it, but that's not the point of this issue

**Comment by rubensworks** [comment]
Any chance you might be able to retrieve the NBT tag of that tank? Something probably went wrong in there.

Also, try restarting the server/client, that might help.

**Comment by Twisted-Code** [comment]
@rubensworks I restarted and nothing fixed. I'll grab the tags if it happens again. As it is, I've already  deleted the item. Might still be in one of my backups of the world, but it seems to happen often enough that I don't feel like digging if I don't have to

**Comment by rubensworks** [comment]
Yeah, I can't reproduce this at all.

Please let me know once you figured out what exactly causes it, so I can reproduce (and fix) it.

**Comment by Twisted-Code** [comment]
@rubensworks sorry, I would probably have the data tags, but I got tired of the tanks breaking and the issue lost priority as I switched to playing a different game. I'm going to go out of my way right now to test it though, so hopefully I'll have some new information for you within the hour

**Comment by Twisted-Code** [comment]
Or not. Seems I'm having trouble reproducing it now. huh. I'll... I'll let you know

**Comment by Twisted-Code** [comment]
I figured it out: Vampirism blood. It looks a lot like any other blood and, although this feature is *undocumented*, gets placed in any available fluid containers (on the hotbar) when a vampire player sucks more from an animal than their blood/"hunger" bar can hold. I'm not sure if that's what happened before because, as mentioned before, I deleted the old tanks, but it's the best explanation I can come up with. Do dark tanks have any indicator of what mod added the fluid in them? I don't remember seeing Vampirism anywhere on the tooltip.

**Comment by rubensworks** [comment]
No, Dark Tanks don't show the fluid's mod in the tooltip.

So I guess this issue can be closed then?

**Comment by Twisted-Code** [comment]
yeah, close it.  About the tooltip: if your the tooltip on your tanks shows the source mod for Biomes o' Plenty blood (and in the version I'm using, it does), I'm not sure why I wouldn't have seen it on vampirism's blood, other than simply overlooking it. Soooo... my bad 
Side note to all of that: I came back to the issue tracker more so because I found a minor rendering bug i wanted to report than to check on this. You know your bottomless water buckets? seems their sprite will change to a normal water bucket under certain circumstances (but the item is still a reusable bucket. Just need to refresh the render). I'll explain in more detail on its own issue, so just look for that in a few minutes.
